### ISSUE TITLE
Configuration via package.json not supported?

### ISSUE THREAD

**Issue by developit** [sindresorhus/atom-linter-xo#1]
Hi there,

It's good to see this coming down the pipe, since XO (+ `esnext`) matches my preferred coding style. 
I have configured my project to set the `esnext` flag via `package.json`, and it seems to work when invoking `xo` manually - but Atom is complaining about the first line of my library, which is a `const` declaration (`Unexpected token const (undefined)`).

I noticed this plugin is just using xo's `lintText` method - perhaps that doesn't allow it to account for rules provided anywhere outside a given file?

**Comment by sindresorhus** [comment]
Thanks for the bug report. Just an oversight by me. Fixed now :)

**Comment by davej** [comment]
I'm having a similar problem.

```json
  "eslintConfig": {
    "extends": "xo-space",
    "env": {
      "mocha": true
    }
  }
```

I'm using xo-space config (and it's installed in local `node_modules`) but atom is still giving me an error when I use spaces. I also tried adding `"space": 2` but it made no difference.

<img width="614" alt="screen shot 2016-01-16 at 21 13 28" src="https://cloud.githubusercontent.com/assets/26971/12374536/1c999e4e-bc96-11e5-827d-fe028b1597c7.png">

**Comment by sindresorhus** [comment]
`"eslintConfig"` → `"xo"`. See: https://github.com/sindresorhus/xo#config

**Comment by davej** [comment]
Thanks!

**Comment by sindresorhus** [comment]
But in this case you could just use the XO `space: true` option.
